src/it/java/net/spy/memcached/util/ConnectionUtilLinuxImpl.java

```diff
@@ -152,5 +158,24 @@ public void removeLocalMemcachedServer(String cmdName, int...ports) {
 		return params;
 	}
 
+	/**
+	 * Params:
+	 * username = memcached_user
+	 * memory limit = 4MB
+	 * enable dynamic reports for 'stats sizes' command
+	 * enable TLS/SSL with certificate and key
+	 * 
+	 * @return
+	 */
+	@Override
+	public Map<String,String> getMemcachedTlsParams() {
```

Comment:
Not needed to add a new method. I would conditionally add in the TLS parameters inside `getMemcachedParams()`